lib/text_cleanup/test/job_log.t/run.t

```diff
@@ -0,0 +1,64 @@
+  $ dune exec text_cleanup -- log | sed 's/\$/%/g' | sed 's/\\n/\n/g' 
+  "~~~ Preparing working directory
+  % cd /home/emile/.buildkite-agent/builds/emile-tarides-1/monorobot-test-org/monorobot-test-pipeline
+  Pseudo-terminal will not be allocated because stdin is not a terminal.
+  
+  # Host \"github.com\" already in list of known hosts at \"/home/emile/.ssh/known_hosts\"
+  % git clean -ffxdq
```

Comment:
```suggestion
  $ git clean -ffxdq
```
Do we want to use dollar signs on the terminal?

Reply:
nvm, i just saw it gets changed 👌

Reply:
Yeah, if I don't change it, it gets interpreted as new cram command when trying to run the tests a second time.